tests/Microsoft.Fx.Portability.Tests/BreakingChangeParserTests.cs

```diff
@@ -191,6 +193,15 @@ public void BreakingChangeMultipleLinks()
             ValidateParse(GetBreakingChangeMarkdown("MultipleBugLinks.md"), expected);
         }
 
+        [Fact]
+        public void CommentsInCategories()
```

Comment:
:suspect: my bad, this test is redundant. Can you please remove it?

Reply:
Done!